plugin/index.js

```diff
@@ -4,28 +4,24 @@
  * @copyright jQuery Foundation and other contributors, https://jquery.org/
  * MIT License
  */
-"use strict";
 
 //------------------------------------------------------------------------------
 // Requirements
 //------------------------------------------------------------------------------
 
-const Generator = require("yeoman-generator");
-const { mkdirSync } = require("fs");
-const validators = require("../lib/validators");
+import Generator from "yeoman-generator";
+import { mkdirSync } from "node:fs"; // eslint-disable-line node/no-missing-import -- https://github.com/mysticatea/eslint-plugin-node/issues/275
```

Comment:
Can we move the comment to shorten line length?

```suggestion
// eslint-disable-next-line node/no-missing-import -- https://github.com/mysticatea/eslint-plugin-node/issues/275
import { mkdirSync } from "node:fs";
```

Reply:
Updated.